src/components/CheckableInput/CheckableInput.tsx

```diff
@@ -16,7 +16,7 @@ export type Props = {
    * Whether the input element should display in indeterminate state.
    */
   indeterminate?: boolean;
-} & HTMLProps<HTMLInputElement>;
+} & Omit<HTMLProps<HTMLInputElement>, "type">;
```

Comment:
Omitting `type` here otherwise `inputType` would get overwritten if you included both `type` and `inputType` props